### DISCUSSION TITLE
Is it possible for users to add custom activity text when entering time?

### DISCUSSION THREAD

**Discussion by jjf539** [kimai/Paid-plugins#145] [Questions & Answers]
Hi! I am in the process of testing out Kimai for usage at my small company and so far it is looking great! Thank you for all the work on this project. One big question I have is if it is possible for users to add custom text for the activity field when entering time. What I mean is instead of a user being able to select only a predefined activity, that they could also enter a blurb for specifically what they worked on that day for a given project. For example a user makes a new time entry for a project and for an activity they input "Fixed bug XYZ, begin investigation bug ABC" and the application accepts/creates a new activity for the project. We have found in our current timekeeping flow this is beneficial for going back and performing quick reviews/audits before invoicing.  

I see there is a custom fields plugin, but it doesn't quite sound like what I am looking for. I also see that from the My Times section you can add a description, but that is not quite it either.  

Let me know if I can provide any clarifications. Essentially I am just curious:

1. If this is currently possible (either by default or through a plugin)?
2. Is that a feature that could be implemented in the feature if it is not possible?

Thanks!

**Comment by kevinpapst** [comment]
Wrong repository, next time please post at https://github.com/kimai/kimai

1. No
2. No, I have no plans to add such a feature currently and money won't help. Use description field or tagging

**Comment by pbksol** [comment]
That field is not meant for the case you are describing. As it will completely clutter your system. Create Activities which can be used without any relation to a customer or Ticket. Keep it as simple as possible and as flexible as possible.

You can add your own fields using the custom fields plugin and add as much information as you wish. You might also use the existing description field to fill in all the things done by the technician.

We did add custom fields to enter a relating ticket id, internal notes and so on.

**Comment by jjf539** [comment]
Thank you guys for the incredibly quick responses and info as to why this field would not work for my use case!
